i18n/en_GB/data/use_of_funds_content.json

```
@@ -92,5 +92,5 @@
 		}
 	},
 
-	"closingParagraph" : "The annual plans of Wikimedia Deutschland and the Wikimedia Foundation provide a detailed insight into our <a href=\"https://meta.wikimedia.org/wiki/Wikimedia_Foundation_Annual_Plan/2025-2026\">international activities</a> and our <a href=\"https://meta.wikimedia.org/wiki/Wikimedia_Deutschland/Plan_2024/Budget_2024\">activities in Germany</a>."
+	"closingParagraph" : "The annual plans of Wikimedia Deutschland and the Wikimedia Foundation provide a detailed insight into our <a href=\"https://meta.wikimedia.org/wiki/Wikimedia_Foundation_Annual_Plan/2025-2026\">international activities</a> and our <a href=\"https://meta.wikimedia.org/wiki/Wikimedia_Deutschland/Plan_2025\">activities in Germany</a>."
```

```suggestion
	"closingParagraph" : "The annual plans of Wikimedia Deutschland and the Wikimedia Foundation provide a detailed insight into our <a href=\"https://meta.wikimedia.org/wiki/Wikimedia_Foundation_Annual_Plan/2025-2026\">international activities</a> and our <a href=\"https://meta.wikimedia.org/wiki/Wikimedia_Deutschland/Plan_2025/en\">activities in Germany</a>."
```

We should link to the English version here, right?

---

@Abban Yes, it's a very good catch. Thanks a lot. I have updated the `EN` file now.